packages/component-test-utils-react/src/methods/html.js

```diff
@@ -1,3 +1,21 @@
+function buildPropsString(props, reactEl) {
+  return props && props.length !== 0 ?
+    ` ${props
+      .map(key => {
+        const keyString = key === 'className' ? 'class' : key;
+
+        // If props is a function
```

Comment:
Instead of that comment and check (I think you would rely a lot on this), I suggest a `isFunction` function 😄